content/en/docs/instrumentation/net/shim.md

```diff
@@ -242,6 +244,36 @@ In the preceding example, `childSpan` is ended because the scope of the `using`
 block is explicitly defined, rather than scoped to `DoWork` itself like
 `parentSpan`.
 
+## Creating independent Spans
+
+The previous examples showed how to create
+[Spans](/docs/concepts/signals/traces/#spans-in-opentelemetry) that follow a
+nested heirarchy. In some cases, you'll want to create independent Spans that
+are siblings of the same root rather than being nested.
+
+```csharp
+public static void DoWork(Tracer tracer)
+{
+    using var sibling1 = tracer.StartSpan("sibling1");
+    using var sibling2 = tracer.StartSpan("sibling2");
+
+    // Both 'sibling1' and 'sibling2' share the same parent
```

Comment:
how ?

Reply:
I think sibling spans should never overlap.

This is the pattern I would recommend:

```
using (var parent = tracer.StartSpan("parent")
{
    using (var child1 = tracer.StartSpan("child1")
    {
    }

    using (var child2 = tracer.StartSpan("child2")
    {
    }
}
```

Reply:
> how ?

In this case I suppose this isn't true because as-written there's no explicit parent, and so both would be root spans. But in general they will share whatever the parent is.

> I think sibling spans should never overlap.

Fair enough, good feedback, will add!